app/controllers/hyrax/file_sets_controller.rb

```diff
@@ -254,12 +225,6 @@ def empty_file?(file)
         (file.respond_to?(:tempfile) && file.tempfile.size == 0) || (file.respond_to?(:size) && file.size == 0)
       end
 
-      # this is provided so that implementing application can override this behavior and map params to different attributes
```

Comment:
I don't disagree with any of that, @atz, but I wonder how removing the code in question helps this PR.

Or maybe we're not as concerned with this specific override capability in Hyrax as we were in Sufia? Thoughts welcome from @jcoyne, the original author of this method.